component/src/governance/component.rs

```diff
@@ -34,8 +34,5 @@ impl Component for Governance {
         execute::enact_pending_parameter_changes(state)
             .await
             .expect("failed to enact parameter changes");
-        execute::apply_proposal_refunds(state)
-            .await
-            .expect("failed to apply proposal refunds");
```

Comment:
We no longer need to apply pending proposal refunds, because they are now self-minted.